swarm/src/connection/pool.rs

```diff
@@ -195,6 +196,8 @@ struct PendingConnection<THandler> {
     endpoint: PendingPoint,
     /// When dropped, notifies the task which then knows to terminate.
     abort_notifier: Option<oneshot::Sender<Void>>,
+    /// The moment we became aware of this possible connection, useful for timing metrics.
+    creation: Instant,
```

Comment:
```suggestion
    accepted_at: Instant,
```

I would suggest using the TCP nomenclature. What do you think?